package.json

```diff
@@ -22,6 +22,7 @@
   "packageManager": "yarn@3.2.3",
   "private": true,
   "scripts": {
+    "g:install": "yarn install && husky",
```

Comment:
Can we do this with `postinstall` instead?

Reply:
Good idea!